kv/dist_sender.go

```diff
@@ -425,7 +425,15 @@ func (ds *DistSender) getDescriptors(
 func (ds *DistSender) sendSingleRange(
 	ctx context.Context, ba roachpb.BatchRequest, desc *roachpb.RangeDescriptor,
 ) (*roachpb.BatchResponse, *roachpb.Error) {
-	log.Trace(ctx, fmt.Sprintf("sending RPC to [%s, %s)", desc.StartKey, desc.EndKey))
+	// Hack: avoid formatting the message passed to Span.LogEvent for
```

Comment:
`TODO` added. I think the pretty-keys are more useful in traces so you don't have to go looking up what the replica ID refers to.